charts/rancher-logging/104.1.0-rc1+up4.8.0/templates/_generic_logging.yaml

```diff
@@ -7,6 +7,48 @@ metadata:
 {{ include "logging-operator.labels" . | indent 4 }}
 spec:
   controlNamespace: {{ .Release.Namespace }}
+<<<<<<< HEAD
```

Comment:
bit of a merge conflict left over here.

Reply:
Rip: thanks for catching. This is what I get for trying to take a shortcut :shrug: